components/new-contribution-flow/index.js

```diff
@@ -71,6 +71,17 @@ const stepsLabels = defineMessages({
   },
 });
 
+const accountCategoriesQuery = gqlV2/* GraphQL */ `
+  query ContributionFlowCategoriesAccountQuery($collectiveSlug: String!) {
+    account(slug: $collectiveSlug, throwIfMissing: false) {
+      id
+      legacyId
+      slug
+      categories
+    }
+  }
+`;
```

Comment:
I think this GraphQL query can be avoided, we should make sure `categories` is available right away in `stepProfile`.